addons/website_event/static/tests/tours/tickets_questions.js

```diff
@@ -6,7 +6,7 @@ registry.category("web_tour.tours").add('test_tickets_questions', {
     url: '/event',
     steps: () => [{
     content: "Click on the Design Fair event",
-    trigger: 'article:contains("Design Fair New York")',
+    trigger: 'div:contains("Design Fair New York")',
 }, {
     content: "Check Register button is disabled when no ticket selected",
     trigger: 'button:disabled:contains("Register")',
```

Comment:
Here indeed you will want to add a step where you click on the register button that opens the new modal.

Then keep this step to check the actual 'register' button in the modal is disabled. And globally readjust the tour so that the steps match the new layout, but I believe most of the tour should still work since it tests the other modal and some pages you haven't touched yet afaict